test/evergreen/gcp_auth.json

```diff
@@ -0,0 +1,13 @@
+{
+    "type": "service_account",
+    "project_id": "axial-radius-373703",
+    "private_key_id": "f4628fbbe1227933d8909320b8711d873961205b",
+    "private_key": "hi",
```

Comment:
I assume this is still in progress, but we'll need to change this back to `${privateKey}` before merging

Reply:
Hi Andrew,
Yes this was a mistake, I have updated it now: e6d7e5b6681cf06055085ce7786051d0ddafc7bb